libethcore/Transaction.cpp

```diff
@@ -130,6 +133,14 @@ Address const& TransactionBase::sender() const
 	return m_sender;
 }
 
+SignatureStruct const& TransactionBase::signature() const
+{ 
+	if (!m_vrs)
```

Comment:
What do you mean by null? Returning default-constructed `SignatureStruct` would mean `[0, 0, 0]` signature which is now considered a special-case valid signature (different thing from not having `m_vrs` at all)

So I think throwing is fine, since this method shouldn't be called for unsigned transactions at all.